server2/Arcadia.Assistant/Arcadia.Assistant.AppCenterBuilds/AppCenterNotification.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Arcadia.Assistant.AppCenterBuilds
+{
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    using Employees.Contracts;
+
+    using Microsoft.Extensions.Logging;
+
+    using Notifications.Contracts;
+    using Notifications.Contracts.Models;
+
+    public class AppCenterNotification : IAppCenterNotification
+    {
+        private readonly string clientName;
+        private readonly ILogger logger;
+        private readonly INotifications notifications;
+
+        public AppCenterNotification(string clientName, INotifications notifications, ILogger logger)
```

Comment:
move client name variable in here, then you can use DI to pass this instance in AppCenterBuilds, instead of creating it manually

Reply:
Done.